### DISCUSSION TITLE
"Failed to launch em desktop application on __init__" issue

### DISCUSSION THREAD

**Discussion by fcogalarza** [ansys/pyaedt#5353] [Q&A]
Hi,

I'm getting the error **_"Failed to launch em desktop application on __init__"_** when starting a new project on AEDT.  The issue happens during PyAEDT initialization when running form new standalone application.  I'm using the following software versions:
- Python = 3.12
- PyAEDt API = 0.11.2
- AEDT  = 2024R2
- Windows

The code I'm using to test the is very simple, and I'm launching this from python console running on poweshell
```python
from ansys.aedt.core import Hfss
hfss = Hfss()
```  
This launch AEDT application but gets stuck for some time to finally fail showing the following error messages:
```
PyAEDT INFO: Python version 3.12.6 (tags/v3.12.6:a4a2d2b, Sep  6 2024, 20:11:23) [MSC v.1940 64 bit (AMD64)]
PyAEDT INFO: PyAEDT version 0.11.2.
PyAEDT INFO: Initializing new Desktop session.
PyAEDT INFO: Log on console is enabled.
PyAEDT INFO: Log on file C:\Users\fgalarza\AppData\Local\Temp\pyaedt_fgalarza_5c5b06ce-08e8-4ddd-ac26-54b5e69ad4d3.log is enabled.
PyAEDT INFO: Log on AEDT is enabled.
PyAEDT INFO: Debug logger is disabled. PyAEDT methods will not be logged.
PyAEDT INFO: Launching PyAEDT with gRPC plugin.
PyAEDT INFO: New AEDT session is starting on gRPC port 50611
PyAEDT INFO: AEDT installation Path C:\Program Files\AnsysEM\v242\Win64
PyAEDT ERROR: **************************************************************
PyAEDT ERROR:   File "<stdin>", line 1, in <module>
PyAEDT ERROR: Failed to launch em desktop application on __init__
PyAEDT ERROR: **************************************************************
PyAEDT ERROR: **************************************************************
PyAEDT ERROR:   File "<stdin>", line 1, in <module>
PyAEDT ERROR: __init__() should return none, not 'bool' on __init__
PyAEDT ERROR: **************************************************************
``` 

I've also tried different combinations of software versions including Python 3.10,  AEDT 2023R2 and API 0.9.0. I'll appreciate any comment suggestion to debug. Thanks, in advance.

**Comment by Samuelopez-ansys** [comment]
Hi @fcogalarza,

This is not expected. Could you try to pass the version explicitly? 

app = Hfss(version=2024.2)

If you go to your temp directory (%temp), you should find a pyaedt log file, could you please share it?

For Python 3.10 and 2024.2, it should work, are you using any proxy?

Please see if this could help:

https://aedt.docs.pyansys.com/version/stable/Getting_started/Troubleshooting.html#check-the-grpc-interface
